### ISSUE TITLE
Why are there no "obama.json" and "obama.pth" files after the training is completed? These two files are required for inference after training, but they were not generated even though the training was successful.

### ISSUE THREAD

**Issue by shiyueGG** [ashawkey/RAD-NeRF#55]
Why are there no "obama.json" and "obama.pth" files after the training is completed? These two files are required for inference after training, but they were not generated even though the training was successful.

./data/
├──.mp4 # original video
├──ori_imgs # original images from video
│ ├──0.jpg
│ ├──0.lms # 2D landmarks
│ ├──...
├──gt_imgs # ground truth images (static background)
│ ├──0.jpg
│ ├──...
├──parsing # semantic segmentation
│ ├──0.png
│ ├──...
├──torso_imgs # inpainted torso images
│ ├──0.png
│ ├──...
├──aud.wav # original audio
├──aud_eo.npy # audio features (wav2vec)
├──aud.npy # audio features (deepspeech)
├──bc.jpg # default background
├──track_params.pt # raw head tracking results
├──transforms_train.json # head poses (train split)
├──transforms_val.json # head poses (test split)

**Comment by iboyles** [comment]
I agree I have just finishing training but it won't inference correctly. I need the colab to be updated for inferencing with a custom trained model and more documentation.

**Comment by hg111** [comment]
@shiyueGG @iboyles did you ever figure out why the obama.json and obama.pth aren't generated?

**Comment by iboyles** [comment]
you probably ran out of compute time during proccessing, I think I ran this on google colab also may not just work in jupyter environment.

**Comment by iboyles** [comment]
The first processing steps are compute heavy and take a few hours.
